contrib/test_jenkins.sh

```diff
@@ -1252,12 +1252,19 @@ run_tests() {
 	# all are running mpi tests
 	run_mpi_tests
 
+	args=""
+	
 	if module_load dev/jdk && module_load dev/mvn
 	then
-		../contrib/configure-devel --prefix=$ucx_inst --with-java
-	else
-		../contrib/configure-devel --prefix=$ucx_inst
+		args+=" --with-java"
 	fi
+
+	if have_cuda
+	then
+		args+=" --with-iodemo-cuda"
+	fi
```

Comment:
let's add ./configure to run_io_demo func and --with-iodemo-cuda usage inside run_io_demo

Reply:
It means `run_io_demo` will rebuild everything (call `$MAKEP` and `$MAKEP install` after `./configure`). Are you sure that's what we want?

Reply:
yes, we cannot assume whatever step was done before run_io_demo did not reconfigure without --with-iodemo-cuda